models/games.models.js

```diff
@@ -35,3 +35,26 @@ exports.selectReviews = () => {
     return rows;
   });
 };
+
+exports.fetchCommentsByReviewId = (id) => {
+  const psqlQueryReview = `SELECT * FROM reviews WHERE review_id = $1`;
+  return db
+    .query(psqlQueryReview, [id])
+    .then((reviewResult) => {
+      const reviewRowCount = reviewResult.rowCount;
+      return reviewRowCount;
+    })
+    .then((reviewRowCount) => {
+      const psqlQuery = `SELECT * FROM comments WHERE review_id = $1 ORDER BY comments.created_at DESC`;
+      return db.query(psqlQuery, [id])
+      .then((result) => {
+        if (reviewRowCount === 0) {
+          return Promise.reject({
+            status: 404,
+            msg: `comments not found!`,
+          });
+        }
+        return result.rows;
+      });
```

Comment:
This could be moved to the .then block above - at the moment, you've got nested .thens here which will potentially mess up your single .then chain.